lib/src/view/menu.dart

```diff
@@ -233,7 +233,7 @@ really want to exit without saving.'''
       DropdownDivider({'key': 'divider-save'}),
       (MenuFormFile()
         ..id = 'import-cadnano-form-file'
-        ..accept = '.json'
+        ..accept = '.sc'
```

Comment:
This line is incorrect. The `accept` field should be `'.json'` since that's the file extension for cadnano files.

Reply:
Yep, I recently made that change but was not able to push it due to failing the "verify formatting" part of unit test checks. Once I get that working, I will push all the updated code with this change and with the other stuff (pop-up, etc.). Thanks for letting me know.

Reply:
@dave-doty I pushed the updated changes. Please let me know if I still need to adjust anything.